
uint8 minY
uint8 maxY